framework/src/operators/packet_batch.rs

```diff
@@ -215,7 +218,7 @@ impl PacketBatch {
                 if ret == 0 {
                     Ok(())
                 } else {
-                    Err(())
+                    Err(NetBricksError::FailedToFreeMBuf.into())
```

Comment:
not sure what `mbuf_free_bulk` returns. maybe keep the return code anyway?

Reply:
it returns 0 or 1. but yeah, I can pass that through.

Reply:
fixing this too. derp

Reply:
oh just 0 or one. hmmm. not as useful then. but whatever.